win32ss/user/ntuser/window.c

```diff
@@ -2394,7 +2394,7 @@ co_UserCreateWindowEx(CREATESTRUCTW* Cs,
 
    if (Window->spwndNext || Window->spwndPrev)
    {
-      ERR("Window 0x%p has been linked too early!\n", Window);
+      ERR("Window 0x%p has been linked too early! (%wZ)\n", Window, ClassName);
```

Comment:
Not sure it is meaningful to show the class name here for that error.

Reply:
Reverted.